tests/take_test.py

```diff
@@ -0,0 +1,112 @@
+import random
+from typing import NoReturn
+
+from lynx.common.actions.action import Action
+from lynx.common.actions.take import Take
+from lynx.common.object import Object
+from lynx.common.scene import Scene
+from lynx.common.vector import Vector
+from lynx.common.enitity import Entity
+
+
+class TestTakeSerialization:
+    expected_serialization_take = '{"type": "Take", "attributes": {"object_id": 1, "position": {"x": 1, "y": 0}}}'
+
+    def test_success_serialization(self) -> NoReturn:
+        serialized_take = Take(object_id=1, position=Vector(1, 0)).serialize()
+
+        assert self.expected_serialization_take == serialized_take
+
+    def test_success_deserialization(self):
+        expected_take = Take(object_id=1, position=Vector(1, 0))
+
+        deserialized_take = Entity.deserialize(self.expected_serialization_take)
+
+        assert expected_take == deserialized_take
+
+
+class TestTakeApply:
+    def test_apply_positive(self):
+        expected_scene = Scene()
+        expected_dummy_object = Object(id=1, name="dummy", position=Vector(5, 5), inventory={"Wood": 1})
+        expected_dummy_take = Take(position=Vector(5, 6), object_id=1)
+        expected_scene.add_entity(expected_dummy_object)
+        expected_scene.add_entity(expected_dummy_take)
```

Comment:
No need for adding `Action` entities when not comparing the whole `Scene`
Same in other test classes